cluster/charts/rook/values.yaml.tmpl

```diff
@@ -26,3 +26,6 @@ logLevel: INFO
 ## If true, create & use RBAC resources
 ##
 rbacEnable: true
+
+# Rook Agent toleration. Choose between NoSchedule, PreferNoSchedule and NoExecute:
+# agentToleration: NoSchedule
```

Comment:
Wouldn't it be better to move this into an "array" named `agent` so when there will be new options later on?
```
agent:
  toleration: NoSchedule
  newOption: false
```
Instead of having `agentNewOption: false`?

Reply:
since there is only one variable for the agent, perhaps it is ok. But it doesnt hurt to start this now. Sounds good.